assignments/objects.js

```diff
@@ -17,29 +17,85 @@ const example = {
   "gender": "F"
 }
 
+
 // Write your intern objects here:
 
+const internMitzi = {
+  id: 1,
+  email: "mmelloy0@psu.edu",
+  name: "Mitzi",
+  gender: "F" 
+}
+
+const internKennan = {
+  id: 2,
+  email: "kdiben1@tinypic.com",
+  name: "Kennan",
+  gender: "M",
+  speak:"Hello, My name is Kennan"
```

Comment:
the property speak here, could be a method, like so, 
`speak: function(){console.log('Hello, My name is ${this.name}');};`.